docs/tooling/docker_private_network.md

```diff
@@ -1 +1,64 @@
 # Docker private network
+
+We provide a tool at `tools/docker-network` with which a local test network can be set up locally with docker. 
+ 
+![Docker network](../../images/docker-network.png)
+
+
+## How to use the tool
+
+In the docker network run for example
+```
+./run.sh 5
```

Comment:
`./run.sh 5 1`, the second argument is for grafana
* default or `0`: disable
* `1`: enable

Reply:
done with reference to the doc where grafana is described in more details